src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -528,7 +524,7 @@ private static void AddProjectReferences(PackageSpec spec, IEnumerable<IMSBuildI
             {
                 // If no frameworks were given, apply to all
                 var addToFrameworks = frameworkPair.Item1.Count == 0
-                    ? aliasGroups.Keys.ToList()
+                    ? aliasGroups.Keys.AsList()
```

Comment:
This change is unnecessary. There's no functional change here.
LINQ generates enumerates which had to be converted to List